main/ui/src/main/java/org/cryptomator/ui/health/HealthReportWriteTask.java

```diff
@@ -30,47 +35,53 @@
 			**************************************
 			Analyzed vault: %s (Current name \"%s\")
 			Vault storage path: %s
-				
 			""";
-	private static final String REPORT_CHECK_SUCCESS = "\tCheck %s successful. Results:\n";
-	private static final String REPORT_CHECK_RESULT = "\t\t %s - %s\n";
-	private static final String REPORT_CHECK_CANCELED = "\tCheck %s canceled.\n";
-	private static final String REPORT_CHECK_FAILED = "\tCheck %s failed.\n";
+	private static final String REPORT_CHECK_HEADER = """
+			   
+			   
+			Check %s
+			------------------------------
+			""";
+	private static final String REPORT_CHECK_RESULT = "%s - %s";
```

Comment:
```suggestion
	private static final String REPORT_CHECK_RESULT = "%8s - %s";
```

so we can drop `severityToString (...)` but keep consistent formatting in the log

Reply:
(or `%-8s` for left-aligned severity)